rfc3164/rfc3164_test.go

```diff
@@ -35,24 +36,25 @@ func (s *Rfc3164TestSuite) TearDownTest(c *C) {
 
 func (s *Rfc3164TestSuite) TestParser_Valid(c *C) {
   buff := []byte("<34>Oct 11 22:14:15 mymachine very.large.syslog.message.tag[17155]: 'su root' failed for lonvick on /dev/pts/8")
-
+ 
   p := NewParser(&buff)
+  p.TimeFunction = nil
   expectedP := &Parser{
     buff:   buff,
     cursor: 0,
     l:      len(buff),
+    TimeFunction: nil,
   }
 
   c.Assert(p, DeepEquals, expectedP)
 
+  p.TimeFunction = octTestDate
   err := p.Parse()
   c.Assert(err, IsNil)
 
-  now := time.Now()
-
   obtained := p.Dump()
   expected := syslogparser.LogParts{
-    "timestamp": time.Date(now.Year(), time.October, 11, 22, 14, 15, 0, time.UTC),
+    "timestamp": time.Date(2015, time.October, 11, 22, 14, 15, 0, time.UTC),
```

Comment:
Will this still pass in 2016?  (if not, make sure to update the other hard coded years well)